src/applications/vaos/utils/data.js

```diff
@@ -40,21 +41,22 @@ export function transformFormToVARequest(state) {
   const facility = getChosenFacilityInfo(state);
   const data = getFormData(state);
   const typeOfCare = getTypeOfCare(data);
+  const systemId = getSystemFromChosenFacility(state);
 
   return {
     typeOfCare: typeOfCare.id,
     typeOfCareId: typeOfCare.id,
     appointmentType: getTypeOfCare(data).name,
     cityState: {
-      institutionCode: data.vaSystem,
-      rootStationCode: data.vaSystem,
-      parentStationCode: data.vaSystem,
+      institutionCode: data.vaParent,
```

Comment:
These changes are important, because it's the mapping to the backend. I believe this will match legacy.

Reply:
Is it common for `rootStationCode` to be different than `parentStationCode`? I assume this is because a child facility could have its own children?

Reply:
I don't think it's common, but it can happen, and it's the reason for us having to separate the idea of parent and root facilities. There are some non-root facilities that are their own parent and have children.